src/status_im/contexts/wallet/wallet_connect/effects.cljs

```diff
@@ -56,13 +49,6 @@
        (promesa/then on-success)
        (promesa/catch on-fail))))
 
-(rf/reg-fx
- :effects.wallet-connect/fetch-active-sessions
- (fn [{:keys [web3-wallet on-success on-fail]}]
-   (-> (wallet-connect/get-active-sessions web3-wallet)
-       (promesa/then on-success)
-       (promesa/catch on-fail))))
-
```

Comment:
we should use the active sessions as the source of truth for "connected dapps" i think